README.md

```
@@ -458,6 +491,160 @@ Disable cleaning (`cleanScrape: false`) only when:
+   ```bash
+   curl http://localhost:3060/health
+   ```
+
+4. **Test with proper headers:**
+
+   ```bash
+   curl -v -H "Host: localhost:3060" http://localhost:3060/health
+   ```
+
+5. **Check MCP initialization:**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)